### PR TITLE
[Mercure] Fix enable_profiler node position

### PR DESCRIPTION
**Pull request by chalasr**
| Q             | A
| ------------- | ---
| License       | MIT
| Doc issue/PR  | -

### CODE REVIEW

**Comment by dunglas** [comment]
Actually `enable_profiler` should be a `0.3` folder, because it's a new flag, and it breaks the stable version.

File: symfony/mercure-bundle/0.3/manifest.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
```
Comment (dunglas) [inline symfony/mercure-bundle/0.3/manifest.json]: This can be a symlink to the file in `0.1`.

Comment (chalasr) [reply symfony/mercure-bundle/0.3/manifest.json]: fixed

**Comment by stof** [comment]
The validation error is because the master branch has alias 1.0.x-dev, not 0.3.x-dev (which looks wrong to me btw, if you plan multiple 0.x releases before reaching 1.0, with BC breaks between them)

**Comment by stof** [comment]
Also, why 0.3 ? There is no 0.2 release yet

**Comment by dunglas** [comment]
You’re right, the next bundle version is 0.2 (I mixed with the library version). And you’re also right about the branch alias. Both the bundle and the lib must be updated.

**Comment by dunglas** [comment]
This PR can now be merged (the CI will be green now).

**Comment by chalasr** [comment]
Green

**Pull request merged**